bash_scripts/run_sim.sh

```diff
@@ -1,40 +1,68 @@
 #!/bin/bash
 
-#SBATCH -p cpu-long
-#SBATCH -c 1
+#SBATCH -p arm-preempt
+#SBATCH -c 20
 #SBATCH -G 0
-#SBATCH --mem=16000
-#SBATCH -t 1-00:00:00
-#SBATCH -o slurm-%j.out
+#SBATCH --mem=40000
+#SBATCH -t 14-00:00:00
+#SBATCH -o slurm-%A_%a.out  # Output file for each task
+#SBATCH --array=0-3  # Define array size based on total combinations
 
-# This script is designed to run a non-artificial intelligence simulation on the Unity cluster at UMass Amherst.
```

Comment:
I think we want to keep this comment to describe what this bash script is designed to do.